l10n_ch_base_bank/models/invoice.py

```diff
@@ -9,6 +9,9 @@ class AccountInvoice(models.Model):
 
     _inherit = "account.invoice"
 
+    reference_type = fields.Selection(
+        selection_add=[('isr','ISR Reference')])
+
```

Comment:
I have added ``isr`` as a possible ``reference_type`` value (instead of the initial ``_get_reference_type()`` function below) , else tests fail reporting something like "isr is an invalid reference_type"...